tests/bootstrap.php

```diff
@@ -0,0 +1,6 @@
+<?php
+if (class_exists('PHPUnit_Framework_TestCase') === false && class_exists('PHPUnit\Framework\TestCase') === true) {
```

Comment:
Could be simplified to `if (!class_exists('PHPUnit_Framework_TestCase') && class_exists('PHPUnit\Framework\TestCase')) {`